dev_guide/builds.adoc

```diff
@@ -952,20 +952,20 @@ a subdirectory within the archive, and if valid, the builder changes into that s
 
 === Image Source
 
-Additional files can be provided to the build process via images.  Input
-images are referenced in the same way the `From` and `To` image targets
-are defined.  This means both docker images and image stream
-tags can be referenced.  In conjunction with the image, you must provide one or
-more path pairs to indicate the path of the files/directories to copy out of the
-image and the destination to place them in the build context.
-
-The source path can be any absolute path within the image specified.  The destination
-must be a relative directory path.  At build time, the image will be loaded
-and the indicated files/directories will be copied into the context directory
-of the build process.  This is the same directory into which the source repository
-content (if any) is cloned.  If the source path ends in `*/.*` then the content
-of the directory will be copied, but the directory itself will not be created
-at the destination.
+Additional files can be provided to the build process via images. Input images
+are referenced in the same way the `*From*` and `*To*` image targets are
+defined. This means both Docker images and image stream tags can be referenced.
+In conjunction with the image, you must provide one or more path pairs to
+indicate the path of the files/directories to copy out of the image and the
```

Comment:
I was told somewhere that slash is undesirable.  Better something like "files and directories".